config-linux.md

```diff
@@ -147,7 +147,7 @@ In addition to any devices configured with this setting, the runtime MUST also s
 * [`/dev/random`][random.4]
 * [`/dev/urandom`][random.4]
 * [`/dev/tty`][tty.4]
-* [`/dev/console`][console.4]
+* [`/dev/console`][console.4] is setup if terminal is enabled in the config by bind mounting the pseudoterminal slave to /dev/console.
```

Comment:
I still think:

* We [want `/dev/console` unconditionally][1].
* We [don't want to require bind-mounting the slave over this][2].  Let the runtime point `/dev/console` to a place it considers appropriate for [logging system messages][3], which may not be the pseudoterminal slave.  In fact, I'd expect the validation for `/dev/console` to be “[Is the major 5, the minor 1, and the type ‘char’?][4]”, which is not going to be satisfied by bind-mounting the slave.

[1]: https://github.com/opencontainers/runtime-spec/pull/518#issuecomment-240494625
[2]: https://github.com/opencontainers/runtime-spec/pull/518#discussion_r73214575
[3]: https://git.kernel.org/cgit/linux/kernel/git/stable/linux-stable.git/tree/Documentation/devices.txt?id=refs/tags/v4.7.1#n3248
[4]: https://git.kernel.org/cgit/linux/kernel/git/stable/linux-stable.git/tree/Documentation/devices.txt?id=refs/tags/v4.7.1#n210